tests/fixture/tmpnet/network.go

```diff
@@ -542,7 +542,7 @@ func (n *Network) RestartNode(ctx context.Context, log logging.Logger, node *Nod
 // Stops all nodes in the network.
 func (n *Network) Stop(ctx context.Context) error {
 	// Target all nodes, including the ephemeral ones
-	nodes, err := ReadNodes(n.Dir, true /* includeEphemeral */)
+	nodes, err := ReadNodes(n, true /* includeEphemeral */)
```

Comment:
nit you could use a named constant instead of a comment. I like the attention to documenting, because it is indeed not clear what `true` is without using an IDE. I tend to use constants for all these arguments, sometimes even nil variables (i.e. `someName := []byte(nil)`) to have a better call documentation. The worst is having a function call `f(nil, false, nil, true, 0)` and have trouble matching each of the argument by position, this is so cognitively unneeded imo. Anyway, rant over 😄 

```suggestion
	const includeEphemeral = true
	nodes, err := ReadNodes(n, includeEphemeral)
```

Same suggestion for other calls of ReadNodes